src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -241,7 +241,7 @@ define(function (require, exports, module) {
      * @return {string}
      */
     function _resolveUrl(path) {
-        return _server && _server.pathToUrl(path);
+        return _server && path;
```

Comment:
On line [741] (https://github.com/humphd/brackets/pull/184/files#diff-e31c028c519c0302491877c1d9c0bad1R741), when _liveDocument.isRelated(absolutePath) is called, it returns undefined even for related documents. This is because [isRelated] (https://github.com/humphd/brackets/blob/bramble/src/LiveDevelopment/MultiBrowserImpl/documents/LiveHTMLDocument.js#L299) calls on _resolveUrl() and it returns the Blob URL. However, the related scripts and stylesheets are indexed by filename.